README.adoc

```diff
@@ -52,7 +52,10 @@ If you want to build everything and run tests, use the `build` task:
 ./gradlew clean build
 ----
 
-This will build everything and run all tests except the `PulsarFunctionAdministrationIntegrationTests` which requires the _Pulsar RabbitMQ Connector_ to be downloaded prior to running.
+NOTE: Some tests rely on a Docker daemon to test against live services using https://www.testcontainers.org/[Testcontainers].
```

Comment:
**[UNRELATED]** I noticed this was missing the fact that docker is required for all tests (it is mentioned in our Working w/ the Code wiki).